ci/upload_coverage.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-
-if [ -z "$COVERAGE" ]; then
-   echo "coverage is not selected for this build"
-   exit 0
-fi
-
-
-echo "uploading coverage"
```

Comment:
Any reason to remove this echo? Figure it doesn't hurt to keep around to give an indication that we've gotten to this point

Reply:
makes sense, got lost when moving the code